microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+
+* Install [Visual Studio Code](https://code.visualstudio.com) to edit source code.
+
+* Install [Node.js](https://nodejs.org) to use it as a local web server.
 
-*  You don't need to know anything about load performance.  You learn about load performance in this tutorial.
+* You don't need to know anything about load performance.  You learn about load performance in this tutorial.
 
 
 <!-- ====================================================================== -->
 ## Introduction
 
-This is Tony.  Tony is very famous in cat society.  He has built a website so that his fans can learn about his favorite foods.  His fans love the site, but Tony keeps hearing complaints that the site loads slowly.  Tony has asked you to help him speed the site up.
+In this tutorial, you improve the performance of [Margie's travel](https://microsoftedge.github.io/Demos/travel-site/), a fictitious travel website that contains travel images, text descriptions, a few JavaScript-based user interactions, and an interactive map.
 
-![Tony the cat](../media/speed-tony.msft.png)
+The source files for the website are at [MicrosoftEdge / Demos > travel-site](https://github.com/MicrosoftEdge/Demos/tree/main/travel-site). 
 
 
 <!-- ====================================================================== -->
-## Step 1: Audit the site
+## Step 1: Set up the website locally
+
+First, set up the website locally so that you can make changes to it later:
+
+1. Get the website's source code locally: [Download or clone the Demos repo](../sample-code/sample-code.md#download-or-clone-the-demos-repo).
+
+1. Open the folder you just downloaded or cloned in Visual Studio Code.
+
+1. In Visual Studio Code, click **View** > **Terminal** or press `Ctrl` + `` ` ``.
+
+   Visual Studio Code displays the source files in the **Explorer** sidebar, and the **Terminal**:
+
+   ![VS Code, now setup with the source code](./images/vscode-setup.png)
+
+1. In the terminal, type `npx http-server` to start a local web server.
+
+1. In Microsoft Edge, go to http://localhost:8080/travel-site to open the website:
+
+   ![The travel website demo in Microsoft Edge](./images/travel-site.png)
+
+
+<!-- ====================================================================== -->
+## Step 2: Audit the site
 
 Whenever you set out to improve the load performance of a site, always start with an audit.
 
 The audit has two important functions:
 
 *  It creates a **baseline** for you to measure subsequent changes against.
 
-*  It gives you **actionable tips** on what changes have the most impact.
+*  It gives you **actionable tips** on what changes improve performance the most.
+
+#### Establish a baseline
+
+The baseline is a record of how the site performed before you made any performance improvements.
 
-### Setup
+1. In Microsoft Edge, open DevTools by pressing `F12` or `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
```

Comment:
I might re-check hidden comments template at bottom of DevTools Overview .md - my fav way to open DevTools is right-click > Inspect.  Does template show that way?